src/app/component/user/components/habit/all-habits/components/habits-list-view/habits-list-view.component.ts

```diff
@@ -6,7 +6,7 @@ import { Component, OnInit, Input } from '@angular/core';
   styleUrls: ['./habits-list-view.component.scss']
 })
 export class HabitsListViewComponent implements OnInit {
-  @Input() habit: object;
+  @Input() habit;
```

Comment:
Please, add a type. If there is no type provided - create model

Reply:
fixed